wg-manifests/charter.md

```diff
@@ -1,70 +1,50 @@
+- Users can also install individual components without the benefits of the platform, but then they could also just directly fetch them from the WG releases.
+- Synchronize the manifests between working groups and make sure via integration tests that the components work end-to-end together as multi-tenant platform
+- Release tested releases of the Kubeflow platform for downstream consumption
+- We try to be compatible with the popular Kubernetes clusters (Kind, Rancher, AKS, EKS, GKE, ...)
+- We provide hints and experimental examples how a user / distribution could integrate non-default external authentication (e.g. companies Identity Provider) and popular non-default services on his own
+- We in general document the installation of Kubeflow as a platform and / or  individual components including common problems and architectural overviews.
+- There is the evolving and not exhaustive list of dependencies for a proper multi-tenant platform installation: Istio, KNative, Dex, Oauth2-proxy, Cert-Manager, ...
+- There is the evolving and not exhaustive list of applications:  KFP, Trainer, Dashboard, Workspaces / Noteboks, Kserve, Spark, ...
+
+## Cross-cutting and Externally Facing Processes
+
+### With Application Owners
+
+- Aid the application owner in creating manifests (Helm, Kustomize) for his application
+- Aid the application owner regarding security best practices
+- Communicate with the application owner regarding releases and versioning
+
+### With Users / Distribution Owners
+- Distributions are opinionated derivatives of Kubeflow platform/manifests, for example replacing all databases with closed source managed databases from AWS, GKE, Azure, ...
+- A distribution can be created by an arbitrary amount of users / companies in private or in public by deriving from Kubeflow platform/manifests, see the definition above
+- Coordinate with "distribution owners" / users to take part in the testing of Kubeflow releases.
 
 ### Out of scope
 
-This WG is NOT going to:
-- Maintain deployment-specific tools like `kfctl`.
-- Maintain distribution-specific manifests.
-- Decide which applications to include in Kubeflow.
-- Decide which variant of an application to include (e.g., KFP Standalone vs
-  KFP with Istio).
-- Create and maintain one or more Kubeflow distributions.
-- Support configurations with environment-specific requirements, like special
-  hardware, different versions of third-party apps (e.g., Istio, KNative, etc.)
-  or custom OIDC providers.
-- Support and promote a specific deployment tool (e.g., `kfctl`). Opinionated
-  deployment tools can extend the base kustomizations to create manifests that
-  support their methods.
-    - For example, people invested in `kfctl` can create overlays that enable
-      the use of `kfctl`'s parameter substitution, which expects a specific
-      folder structure (`params.env`).
+- We do not support a specific deployment tool (e.g., ArgoCD, Flux)
```

Comment:
This needs to be clarified, what do we mean by "support":

- what is a "deployment tool"?
- in what way are we "not supporting" deployment tools?